CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -806,7 +806,7 @@ CF_EXPORT void _NS_pthread_setname_np(const char *name);
 #endif
 
 #if DEPLOYMENT_TARGET_LINUX
-CF_EXPORT Boolean _CFIsMainThread(void);
+CF_EXPORT int _CFIsMainThread(void);
```

Comment:
... And be removed from CFInternal.h.

Also I think we should leave the result as Boolean.